Test.C

```diff
@@ -581,3 +549,65 @@ void encryptionStore(struct Player *playerPtr, struct Encryption encryptionStock
         return;
     }
 }
+/******************************************************************************
+* Function:    SellingStore
+* Description: Interactive store where player can sell equipment
+* Parameters:  playerPtr, Player, pointer to the player object
+*              equipmentStock, Equipment, array of all possible equipments
+* Return:      void
+******************************************************************************/
+
+void SellingStore(struct Player *playerPtr, struct Equipment equipmentStock[NUM_EQUIPMENT]) {
+    int equipSellSelect;
+    int equipSellAmount;
+    int equipSellTransaction;
+    char equipSellAnswer;
+
+    printf("\n");
+    printf("|----------------------------- Equipment Selling ------------------------------|\n");
+    printf("Equipment that you own:\n");
+    printf(" # | Equipment Name                       | Cost  | Message Amount | Inventory |\n");
+    printf("---|--------------------------------------|-------|----------------|-----------|\n");
+    for(int i=0; i<NUM_EQUIPMENT; i++){
```

Comment:
Space after for per the style guide
```suggestion
    for (int i=0; i<NUM_EQUIPMENT; i++){
```